### ISSUE TITLE
A11y_Deploymentconfigurationtemplate_Usability:Missing keyboard hotkey for "previous" and "next button for keyboard only users

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7946]
GitHubTags: #A11y_AzureDataStudio;#A11yUsable; #A11yTCS;#SQL Azure Data Studio;#DesktopApp;#Win32;#NVDA;
Environment Details:
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit:6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
Steps to Reproduce:
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster” .
5.     Navigate till next and previous button 
Actual:
There were no any keyboard hotkey implementation is available for "previous" and "next" buttons for screen reader as well as screen reader users.
Expected:
There should be keyboard hotkey implementation is available for "previous" and "next" buttons for screen reader as well as screen reader users.
**User Impact:**
keyboard only users finds difficult to access the controls easily as well as finds difficult to navigate respective content again and again.
**Attachment for Reference:**

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
this is for Grade B (Usability), i am going to leave this open until we are planning to get to that level as a whole.

**Comment by alanrenmsft** [comment]
closing as won't fix since this is not accessibility grade c blocker.

**Comment by V-sheman** [comment]
Science this Usability issue at this point of time we can not take any action on it.
